sample_size/multiple_testing.py

```diff
@@ -25,7 +28,15 @@ class MultipleTestingMixin:
     power: float
     variants: int
 
-    def get_multiple_sample_size(self, lower: float, upper: float, depth: int = 0) -> int:
+    def get_multiple_sample_size(
+        self,
+        lower: float,
+        upper: float,
+        depth: int = 0,
+        replication: int = DEFAULT_REPLICATION,
+        epsilon: float = DEFAULT_EPSILON,
+        max_recursion_depth: int = DEFAULT_MAX_RECURSION,
+    ) -> int:
         """
         This method finds minimum required sample size per cohort that generates average power higher than required
 
```

Comment:
I made a mistake in the docstring. It should be 
```
lower: lower bound of sample size search; minimum of each metric's individually calculated sample size
```

Reply:
I got rid of the text after the ";". Those conditions only apply once when the function is initially called